.github/workflows/test_and_deploy.yml

```diff
@@ -0,0 +1,173 @@
+name: Deploy and Test
+
+on:
+  push:
+    branches:
+      - "*"
```

Comment:
Change it to "github_workflow/*" branches rather than only "*"

Change your current branch to github_workflow/blahblah to check the workflow run